src/backend/joanie/badges/providers/obf.py

```diff
@@ -0,0 +1,468 @@
+"""OpenBadgeFactory provider."""
```

Comment:
For your information, during your holidays, we refactor tests architecture to centralize them within the same folder.
You'll have to move this test suite accordingly.
https://github.com/openfun/joanie/pull/128

Reply:
<img src="https://media0.giphy.com/media/xakXSnCpsqZWM/giphy.gif"/>